app/Resources/views/events/event_summary.csv.twig

```diff
@@ -0,0 +1,14 @@
+{% for metric in ['participants', 'pages-created', 'pages-improved', 'edits', 'byte-difference', 'files-uploaded', 'items-created', 'items-improved', 'pages-created-pageviews', 'pages-improved-pageviews-avg', 'pages-using-files', 'file-usage', 'pages-using-files-pageviews-avg', 'new-editors', 'retention'] %}{#
+    #}{% set stat = event.getStatistic(metric) %}{#
+    #}{% if stat is not empty %}{#
+        #}{{ msg(metric, [stat.offset]) }},{#
+        #}{% if stat.value is null %} {% else %}{% if metric == 'byte-difference' and stat.value > 0 %}+{% endif %}{{ stat.value }}{% endif %}
+
+{% endif %}{#
+#}{% endfor %}
+
+{{ msg('event-details') }}
+{{ msg('event-summary') }}, {{ event.displayTitle }}
+{{ msg('from') }}, {{ event.start|date_localize }}
+{{ msg('to') }}, {{ event.end|date_localize }}
```

Comment:
Sorry my example at https://phabricator.wikimedia.org/T205561#4982073 said to use this language. I think we should definitely use the existing `start-date` and `end-date` messages. I doubt Joe will have a problem with tihs.